assets/js/templates/snippet/image.html

```diff
@@ -0,0 +1,6 @@
+<!-- File Button -->
+<div class="control-group">
+  <label class="control-label"
+         for="<%= field_guid %>"><%= label %></label>
+  <button id="<%= field_guid %>" name="<%= field_guid %>">Upload</button>
```

Comment:
I'm not sure if we have a means of translating "Upload" but that could be something. I think it only shows up in the edit UI so maybe it doesn't matter much.